### PR TITLE
EN-4377 - Adding support for customVerificationHandler (to enable automation tests)

### PR DESCRIPTION
**Pull request by dimapx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +137,47 @@ Example below:
...
+
+namespace myUniqueApp
+{
+    public class UniqueVerificationHandler : ICustomVerificationHandler
+    {
+        public void Handle(HttpApplication application, PxContext pxContext, PxModuleConfigurationSection pxConfig)
+        {
+            // Custom verification logic goes here, e.g -
+            // 
+            // if (pxContext.Score >= pxConfig.BlockingScore) // In the case of a high score, present the standard block/captcha page
```
Comment (yaronpx) [inline README.md]: why is this code example commented out?

File: PerimeterXModule/CustomBehavior/ICustomVerificationHandler.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System.Web;
+
+namespace PerimeterX
+{
+    /// <summary>
+    /// Custom verification handler, can be used to replace the default verification behavior 
+    /// to execute custom logic based on the risk score returned by PerimeterX.
+    /// </summary>
+    public interface ICustomVerificationHandler
```
Comment (nitzanpx) [inline PerimeterXModule/CustomBehavior/ICustomVerificationHandler.cs]: Name it IVerificationHandler

**Comment by nitzanpx** [comment]
merge after solving comment :)

File: README.md (hunk 1 above)
Comment (dimapx) [reply README.md]: I can uncomment it (assuming that the comment in the first line above is enough to indicate that this is no more than an example and not necessarily part of the expected custom logic).

File: PerimeterXModule/CustomBehavior/ICustomVerificationHandler.cs (hunk 2 above)
Comment (dimapx) [reply PerimeterXModule/CustomBehavior/ICustomVerificationHandler.cs]: 👍

File: README.md (hunk 1 above)
Comment (yaronpx) [reply README.md]: I don't think the commented out code explains that this is an example anyway. But maybe change the line above to something like `Custom verification logic goes here. The following code is only an example of a possible implementation.`

Comment (dimapx) [reply README.md]: 👍

**Pull request merged**